src/components/layout/footer/main-links.js

```diff
@@ -0,0 +1,227 @@
+                        </Link>
+                    </LinkWrapper>
+                    <LinkWrapper>
+                        <Link to="/contact_us/">{localize('Contact us')}</Link>
+                    </LinkWrapper>
+                    <LinkWrapper>
+                        <Link to="/careers/">{localize('Careers')}</Link>
+                    </LinkWrapper>
+                </LinksCol>
+                <LinksCol>
+                    <LinkWrapper>
+                        <Title>{localize('TRADE')}</Title>
+                    </LinkWrapper>
+                    <LinkWrapper first_child="true">
+                        <Link to="/dtrader/">{localize('DTrader')}</Link>
+                    </LinkWrapper>
+                    <LinkWrapper>
+                        <Link to="/dbot/">{localize('DBot')}</Link>
+                    </LinkWrapper>
+                    <LinkWrapper>
+                        <Link to={is_ppc_redirect ? '/landing/dmt5/' : '/dmt5/'}>
+                            {localize('DMT5')}
+                        </Link>
+                    </LinkWrapper>
+                    <LinkWrapper>
+                        <Link
+                            to="trading"
+                            type="smart_trader"
+                            external="true"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            {localize('SmartTrader')}
+                        </Link>
+                    </LinkWrapper>
+                </LinksCol>
+                {!is_ppc && (
+                    <LinksCol>
+                        <LinkWrapper>
+                            <Title>{localize('TRADE TYPES')}</Title>
+                        </LinkWrapper>
+                        <LinkWrapper first_child="true">
+                            <Link to="/trade-types/margin/">
+                                {localize('Margin trading')}
+                            </Link>
+                        </LinkWrapper>
+                        <Show.NonEU>
+                            <LinkWrapper>
+                                <Link to="/trade-types/options/">
+                                    {localize('Options')}
+                                </Link>
+                            </LinkWrapper>
+                        </Show.NonEU>
+                        <LinkWrapper>
+                            <Link to="/trade-types/multiplier/">
+                                {localize('Multipliers')}
+                            </Link>
+                        </LinkWrapper>
+                    </LinksCol>
+                )}
+                <LinksCol>
+                    <LinkWrapper>
+                        <Title>{localize('MARKETS')}</Title>
+                    </LinkWrapper>
+                    <LinkWrapper first_child="true">
+                        <Link to="/markets#forex">{localize('Forex')}</Link>
```

Comment:
```suggestion
                        <Link to="/markets/forex/">{localize('Forex')}</Link>
```

Reply:
And for the other markets as well. Think we're missing crypto one here. I think because the latest master one is on another file. Please also help to update them to end with trailing slash as well. Thanks!